cmd/cluster.go

```diff
@@ -495,6 +496,8 @@ func installSealedSecrets() {
 	secret, err := cmd.Output()
 	check(err)
 
+	os.MkdirAll(secretsDir, 0755) //if .secrets folder is missing, create
```

Comment:
keep comments above the code they relate to, not inline. 

also since there is no if statement here. perhaps to show the true intention of the code its better to say ensure secrets folder is present. or something along those lines.